examples/app.py

```diff
@@ -117,13 +117,14 @@
             pid_fetcher='recid',
             search_index='testrecords',
             search_type=None,
+            search_factory_imp='invenio_records_rest.query.es_search_factory',
```

Comment:
I'm not sure we should make this the default query parser at it has some downsides like being able to search on anything, even if it's hidden.